get_libwallet_api.sh

```diff
@@ -1,60 +1,66 @@
+if [ ! -d $MONERO_DIR ]; then
+    git clone --depth=1 $MONERO_URL $MONERO_DIR --branch $MONERO_BRANCH --single-branch
 else
-    cd $BITMONERO_DIR;
-    git checkout $BITMONERO_BRANCH
+    cd $MONERO_DIR;
+    git checkout $MONERO_BRANCH
     git pull;
 fi
 
-echo "cleaning up existing bitmonero build dir, libs and includes"
-rm -fr $BITMONERO_DIR/build
-rm -fr $BITMONERO_DIR/lib
-rm -fr $BITMONERO_DIR/include
+echo "cleaning up existing monero build dir, libs and includes"
+rm -fr $MONERO_DIR/build
+rm -fr $MONERO_DIR/lib
+rm -fr $MONERO_DIR/include
 
-mkdir -p $BITMONERO_DIR/build/release
-pushd $BITMONERO_DIR/build/release
+mkdir -p $MONERO_DIR/build/release
+pushd $MONERO_DIR/build/release
 
 if [ "$(uname)" == "Darwin" ]; then
     # Do something under Mac OS X platform        
-    cmake -D CMAKE_BUILD_TYPE=Release -D STATIC=ON -D BUILD_GUI_DEPS=ON -D CMAKE_INSTALL_PREFIX="$BITMONERO_DIR"  ../..
+    cmake -D CMAKE_BUILD_TYPE=Release -D STATIC=ON -D BUILD_GUI_DEPS=ON -D INSTALL_VENDORED_LIBUNBOUND=ON -D CMAKE_INSTALL_PREFIX="$MONERO_DIR"  ../..
 elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
     # Do something under GNU/Linux platform
-    cmake -D CMAKE_BUILD_TYPE=Release -D STATIC=ON -D BUILD_GUI_DEPS=ON -D CMAKE_INSTALL_PREFIX="$BITMONERO_DIR"  ../..
+    PLATFORM="Linux"
+    cmake -D CMAKE_BUILD_TYPE=Release -D STATIC=ON -D BUILD_GUI_DEPS=ON -D CMAKE_INSTALL_PREFIX="$MONERO_DIR"  ../..
 elif [ "$(expr substr $(uname -s) 1 10)" == "MINGW64_NT" ]; then
     # Do something under Windows NT platform
-    cmake -D CMAKE_BUILD_TYPE=Release -D STATIC=ON -D BUILD_GUI_DEPS=ON -D CMAKE_INSTALL_PREFIX="$BITMONERO_DIR" -G "MSYS Makefiles" ../..
+    cmake -D CMAKE_BUILD_TYPE=Release -D STATIC=ON -D BUILD_GUI_DEPS=ON -D INSTALL_VENDORED_LIBUNBOUND=ON -D CMAKE_INSTALL_PREFIX="$MONERO_DIR" -G "MSYS Makefiles" ../..
 elif [ "$(expr substr $(uname -s) 1 10)" == "MINGW32_NT" ]; then
     # Do something under Windows NT platform
-    cmake -D CMAKE_BUILD_TYPE=Release -D STATIC=ON -D BUILD_GUI_DEPS=ON -D CMAKE_INSTALL_PREFIX="$BITMONERO_DIR" -G "MSYS Makefiles" ../..
+    cmake -D CMAKE_BUILD_TYPE=Release -D STATIC=ON -D BUILD_GUI_DEPS=ON -D INSTALL_VENDORED_LIBUNBOUND=ON -D CMAKE_INSTALL_PREFIX="$MONERO_DIR" -G "MSYS Makefiles" ../..
 fi
 
 
-pushd $BITMONERO_DIR/build/release/src/wallet
+pushd $MONERO_DIR/build/release/src/wallet
 make -j$CPU_CORE_COUNT
 make install -j$CPU_CORE_COUNT
 popd
 
 # unbound is one more dependency. can't be merged to the wallet_merged
 # since filename conflict (random.c.obj)
-pushd $BITMONERO_DIR/build/release/external/unbound
-make -j$CPU_CORE_COUNT
-make install -j$CPU_CORE_COUNT
-popd
+# for Linux, we use libunbound shipped with the system, so we don't need to build it
+
+if [ "$PLATFORM" != "Linux" ]; then
+    echo "Building libunbound..."
+    pushd $MONERO_DIR/build/release/external/unbound
+    make -j$CPU_CORE_COUNT
+    make install -j$CPU_CORE_COUNT
+    popd
+fi
```

Comment:
Yes, most likely lines 32-50 will build libunbound (due it's a dependency for some library within libwallet_merged), but it will not copy it into "lib/" until ```make install``` invoked